src/core/SkCpu.cpp

```diff
@@ -46,7 +47,6 @@
         if (abcd[2] & (1<< 9)) { features |= SkCpu::SSSE3; }
         if (abcd[2] & (1<<19)) { features |= SkCpu::SSE41; }
         if (abcd[2] & (1<<20)) { features |= SkCpu::SSE42; }
-
```

Comment:
These types of changes make upstream merging harder. Probably unintended anyway?

Reply:
done.